tsconfig.json

```diff
@@ -14,4 +14,5 @@
     "exclude": [
         "node_modules/"
     ]
-}
\ No newline at end of file
+}
+    
```

Comment:
Line 7 has some whitespaces, delete those whitespaces or tabs, without deleting line 7. Or maybe you have two blank lines, and it is just needed one.